src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -174,23 +174,22 @@ public static void deleteColony(final int id)
                     colonyWorlds.add(world);
                 }
             }
-            Log.getLogger().info("Removing buildings for " + id);
-            for (final AbstractBuilding building : new ArrayList<>(colony.getBuildings().values()))
-            {
+
+            if (canDelete = true) {
+                Log.getLogger().info("Removing buildings for " + id);
+                for (final AbstractBuilding building : new ArrayList<>(colony.getBuildings().values())) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)